torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12286,6 +12296,20 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                DecorateInfo(unittest.expectedFailure, 'TestMeta', 'test_dispatch_meta_outplace', device_type='cuda'),
                DecorateInfo(unittest.expectedFailure, 'TestMeta', 'test_dispatch_symbolic_meta', device_type='cuda'),
            )),
+    OpInfo('view_copy',
+           dtypes=all_types_and(torch.bool, torch.bfloat16, torch.float16, torch.chalf),
+           supports_out=True,
+           supports_forward_ad=True,
+           supports_fwgrad_bwgrad=True,
+           supports_autograd=True,
+           sample_inputs_func=sample_inputs_view_copy,
+           skips=(
+               # view_copy does not support automatic differentiation for outputs with complex dtype
+               DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_dtypes'),
```

Comment:
I'll check, but this test does test does not have a `_{dtype}` suffix, so I didn't think that would work.

Reply:
@bdhirsh we may want to add complex autograd support for all these `_copy` ops.

Reply:
@amjames adding `dtypes=(torch.complex32, torch.complex64, torch.complex128)` to `DecorateInfo ` has no effect I think because the test uses `@ops(ops_and_refs, dtypes=OpDTypes.none)`

Reply:
I think, instead of xfailing the test, it would be better to specify `backward_dtypes` for the OpInfo excluding the complex dtypes and open an issue for complex autograd support.

This way the test can still run checking the other things.

Reply:
adding `backward_dtypes=` does not seem to work in this case, looking at the test, it does not appear to use this: 

```py
include_complex32 = (
            (torch.complex32,)
            if op.supports_dtype(torch.complex32, device_type)
            else ()
        )
```